apps/workers/workflow-workers/.env.example

```
@@ -30,4 +30,12 @@ LANGFUSE_SECRET_KEY=
 # Default: https://cloud.langfuse.com (EU) or https://us.cloud.langfuse.com (US)
-LANGFUSE_BASEURL=https://us.cloud.langfuse.com
\ No newline at end of file
+LANGFUSE_BASEURL=https://us.cloud.langfuse.com
+
+# =============================================================================
+# AGENT CONTAINER IMAGE
+# =============================================================================
+# [Required] Base Docker image for agent containers.
+# - Local dev: issue-to-pr/agent-base:dev (built by `pnpm docker:build:agent-base`)
+# - Production: ghcr.io/youngchingjui/agent-base
+AGENT_BASE_IMAGE=issue-to-pr/agent-base:dev
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)